packages/reports/index.js

```diff
@@ -1,4 +1,8 @@
 'use strict';
+// eslint-disable-next-line node/no-extraneous-require
+let Funnel = require('broccoli-funnel');
+// eslint-disable-next-line node/no-extraneous-require
```

Comment:
Why do we have these disable lints? Did we add `broccoli-merge-trees` & `broccoli-funnel` as dependencies in the package.json?